src/NLog.Web/NLogHttpModule.cs

```diff
@@ -1,56 +1,53 @@
-﻿using System;
+﻿using NLog.Web.Targets.Wrappers;
+using System;
 using System.Web;
 
 namespace NLog.Web
 {
     /// <summary>
-    /// ASP.NET HttpModule that enables NLog to hook BeginRequest and EndRequest events easily.
+    /// ASP.NET IHttpModule that enables AspNetBufferingTargetWrapper proper functioning
     /// </summary>
     public class NLogHttpModule : IHttpModule
     {
-        /// <summary>
-        /// Event to be raised at the end of each HTTP Request.
-        /// </summary>
-        public static event EventHandler EndRequest;
```

Comment:
Can `EndRequest` + `BeginRequest` be restored, since it is a breaking change to remove them.

Reply:
Completed